spec/system/candidate_interface/entering_details/candidate_edits_or_deletes_volunteering_section_with_restructured_work_history_active_spec.rb

```diff
@@ -0,0 +1,105 @@
+require 'rails_helper'
+
+RSpec.feature 'Candidate edits their volunteering section' do
+  include CandidateHelper
+
+  scenario 'Candidate adds or deletes a role after completing the section' do
+    FeatureFlag.activate('restructured_work_history')
```

Comment:
https://github.com/DFE-Digital/apply-for-teacher-training/pull/4213/files#r592392701

... And wrap in a method `given_the_foo_is_bar`

Reply:
I got rid of that change in a later commit